Database/Corrections/cataQuestFixes.lua

```diff
@@ -1667,6 +1667,9 @@ function CataQuestFixes.Load()
         [14324] = { -- Full of Hot Water
             [questKeys.extraObjectives] = {{nil, Questie.ICON_TYPE_EVENT, l10n("Summon the Scalding Water Lord"), 0, {{"object", 195711}}}},
         },
+        [14333] = { -- While You're Here
+            [questKeys.objectives] = {{nil,nil,{49194,nil,Questie.ICON_TYPE_INTERACT}}},
```

Comment:
According to wowhead this quest has two objectives: https://www.wowhead.com/quest=14333/while-youre-here 🤔 

One NPC objective which is 36204 (a "bunny" objective, which Blizzard does for some quests) and also an item objective which are dropped by [this object](https://www.wowhead.com/object=196395/defiled-relic).

You can verify this by running `/dump C_QuestLog.GetQuestObjectives(14333)` in the chat (most likely twice in order to return correct data).

If that holds, the fix would be to add most the monster objectve as well as the item objective to the quest.

Reply:
There is actually only a single objective:
![image](https://github.com/user-attachments/assets/cf4235f9-9a69-427b-a5c3-5185998611e9)